addons/neon/templates/clusterdefinition.yaml

```diff
@@ -95,7 +119,7 @@ spec:
       characterType: safekeeper
       componentDefRef:
         - &storagebrokerRef
-          componentDefName: neon-storagebroker
+          componentDefName: neon-storbroker
```

Comment:
componentDefName here is the name of clusterDefinition.spec.componentDefs[x].name，not the name of ComponentDefinition